documentation/docs/packages/camera-web.md

```diff
@@ -1,13 +1,15 @@
 ---
-sidebar_position: 1
+sidebar_position: 2
 ---
 
 # camera-web
+
```

Comment:
Why did you add these blank lines under every title in the different markdown documents? My guess is that it's an auto-correct thing made by your IDE? Remove all of these blank lines,, there are a lot in the PR

Reply:
Yes it was my auto-correct !